pymobiledevice3/services/crash_reports.py

```diff
@@ -129,22 +129,36 @@ def watch(self, name: str = None, raw: bool = False) -> Generator[str, None, Non
                 else:
                     yield crash_report
 
-    def get_new_sysdiagnose(self, out: str, erase: bool = True, *, timeout: Optional[float] = None) -> None:
+    def get_new_sysdiagnose(self, out: str, erase: bool = True, *, timeout: Optional[float] = None,
```

Comment:
add exact typing to `callback`

Reply:
Noted! let me know if the commit below works!